clang/test/SemaSYCL/intel-reqd-work-group-size-device.cpp

```diff
@@ -1,5 +1,7 @@
-// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -Wno-sycl-2017-compat -fsyntax-only -verify -DTRIGGER_ERROR %s
-// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -Wno-sycl-2017-compat -ast-dump %s | FileCheck %s
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -sycl-std=2017  -Wno-sycl-2017-compat -fsyntax-only -verify -DTRIGGER_ERROR %s
```

Comment:
You don't use `TRIGGER_ERROR `in test anymore. I see its been deleted

Reply:
> You don't use `TRIGGER_ERROR `in test anymore. I see its been deleted

We still need this since we use #ifdef TRIGGER_ERROR on line no 104. if you expand the test, you would be able to see this.

Reply:
Oh my bad. Sorry for confusion!